src/shared/install.c

```diff
@@ -1492,7 +1492,7 @@ static int install_info_symlink_alias(
                 rp = skip_root(paths, i->path);
 
                 q = create_symlink(rp ?: i->path, alias_path, force, changes, n_changes);
-                if (r == 0)
+                if (q < 0)
                         r = q;
```

Comment:
it's going to be 0 initialially. But this is a loop, and as soon as the if check hits once, then r is overriden with q, which might not be 0 anymore... So no, this is not always true...